js/client-battle-tooltips.js

```diff
@@ -408,6 +425,25 @@ var BattleTooltips = (function () {
 		text += '</div></div>';
 		return text;
 	};
+	
+	BattleTooltips.prototype.findBoost = function (pokemon, stat) {
```

Comment:
Y'know, in addition to fixing the errors Travis CI is telling you to fix. For future reference, running `npm test` yourself can save time in terms of this.